lambda_functions/plumbing.py

```diff
@@ -146,7 +146,7 @@ def setup_sns_publishing(boto_session):
         Targets=[
             {
                 'Arn': find_or_create_sns_topic(boto_session),
-                'Id': 'gd2MdSNS',
+                'Id': 'normalizationSNS',
```

Comment:
I wonder if, upon doing a stack update to this new code, this will cause the creation of a second CloudWatch Event Target due to the new Id value. @andrewkrug thoughts?